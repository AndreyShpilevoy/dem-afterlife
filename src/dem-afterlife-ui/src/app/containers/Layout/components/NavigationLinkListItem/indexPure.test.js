/* eslint no-undef: 0, fp/no-unused-expression: 0, fp/no-nil: 0*/

import React from 'react';
import {shallow} from 'enzyme';
import {NavigationLinkListItemPure} from './index';

describe('NavigationLinkListItem Pure', () => {
    const styles = {
        '.default-NavigationLinkListItem-link:hover': 'default-NavigationLinkListItem-link:hover',
        link: 'default-NavigationLinkListItem-link'
    };

    it('component match expected snapshot', () => {
        const navigationLinkItem = {id: 1, title: 'Conference', href: '/', order: 1};
        expect(shallow(<NavigationLinkListItemPure styles={styles} navigationLinkItem={navigationLinkItem}/>)).toMatchSnapshot();
    });
});
