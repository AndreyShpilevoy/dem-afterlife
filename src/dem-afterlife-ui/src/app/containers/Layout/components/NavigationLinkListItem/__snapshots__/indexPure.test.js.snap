// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationLinkListItem Pure component match expected snapshot 1`] = `
<li>
  <withStyles(LinkPure)
    className="default-NavigationLinkListItem-link"
    style={Object {}}
    to="/"
  >
    Conference
  </withStyles(LinkPure)>
</li>
`;
